lib/jxl/dec_modular.cc

```diff
@@ -470,6 +470,7 @@ Status ModularFrameDecoder::ModularImageToDecodedRect(
     return true;
   }
   JXL_DASSERT(rect.IsInside(decoded));
+  JXL_DASSERT(gi.transform.empty());
```

Comment:
Please make these regular JXL_ASSERT instead of JXL_DASSERT, or even JXL_CHECK. There's no reason to only have these safety checks in msan/asan/debug builds but not in release builds; the impact on speed and binary size will be very close to zero and the improvement in security is worth it, imo.

Reply:
Done.